### PR TITLE
added some props for MultiList component along with addon-knobs

### PR DESCRIPTION
**Pull request by divyanshu-rawat**
Added

1. with size
2. with filter
3. Default Selected

Apart from that adding props like With queryFormat requires to make use of  ``` @storybook/addon-knobs ```  I have updated the ``` package.json ``` by adding as a
devDependencies 
``` 
 "@storybook/addon-knobs": "^4.0.4"
```
and also registed it by creating addons.js inside .storybook directory but there is no official documentation of storybook to configure ```addon-knobs``` inside index.js, so I have opened an issue for the same in storybook github repo.

Apart from that, Added props are working fine. 
I have tested them locally and commented out ``` addon-knobs ``` configuration lines as we have to make use of them in future once I know how to configure them for Vue.js.

### CODE REVIEW

**Comment by divyanshu-rawat** [comment]
@bietkul ``` with filter ``` is causing some issue let me update this PR by fixing it 👍

File: stories/index.js [hunk 1]
```diff
@@ -106,6 +123,39 @@ storiesOf('Search Components/DataSearch', module)
...
+  }))
+  .add('With fuzziness as a number', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ fuzziness: 1, showFilter: false}\"/>",
+  }))
+  .add('With fuzziness as AUTO', () => ({
+    components: { BaseDataSearch },
+    template: "<base-data-search :subProps=\"{ fuzziness: AUTO, showFilter: false}\"/>",
+  }))
+   .add('With highlight', () => ({
```
Comment (jyash97) [inline stories/index.js]: For highlight to work, we need to add `v-html` to render raw HTML currently it is displaying something like this `<mark>Harry</mark>`

Comment (divyanshu-rawat) [reply stories/index.js]: @jyash97 yes I agree with that, I tried passing doing this, 

```js
   .add('With highlight', () => ({
    components: { BaseDataSearch },
    template: "<base-data-search rawHtml=\"<span style= 'color: red'>hey</span>\" 
    :subProps=\"{ highlight: true, showFilter: false}\"/>",
  }))
```

and doing this 

```js
  <DataSearch
    className="result-list-container"
    categoryField="authors.raw"
    componentId="BookSensor"
    :dataField="['original_title', 'original_title.search']"
    v-bind='subProps'
    v-on='subEvents'
    v-html='rawHtml'
  />
```

but this replaces `DataSearch` component with `hey`, so you let me know how exactly I can add `v-html`

Comment (divyanshu-rawat) [reply stories/index.js]: @jyash97 I don't think we need to manually inject v-html as a prop, I think it depends upon how the ```DataSearch ``` component is written in vue.js.
So, I guess @bietkul can help us in this.

Comment (bietkul) [reply stories/index.js]: Just sync this branch with master once, it should be resolved.

Comment (divyanshu-rawat) [reply stories/index.js]: @bietkul but intially the branch has no conflicts? have you updated master?

Comment (divyanshu-rawat) [reply stories/index.js]: Just check it is resolved now.

**Pull request merged**